src/components/pages/SignIn.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { SignInApiManager } from "../services/user";
+
+const SignIn = () => {
+  const [user, setUser] = useState({ email: "", password: "" });
+  
+  const handleClick = async function () {
+    const dbRes = await SignInApiManager();
+    let isFound = false;
+    for (let i = 0; i < dbRes.length; i++) {
+      if (user.email === dbRes[i].email) {
+        if (user.password === dbRes[i].password) {
+          alert("yay you are a user");
+          isFound = true;
+        }
```

Comment:
We already check the email and password in the backend, why do it again in the front